### ISSUE TITLE
CI build action integration

### ISSUE THREAD

**Issue by peter279k** [ua1-labs/firetest#6]
As title, to make this testing tool reliable, we should consider integrating CI service.

The available CI build services are as follows:

- Travis CI
- CircleCI
- GitHub Actions

@joshualjohnson, I don't know which services are proper for this repository.

Please refer this issue.

**Comment by joshualjohnson** [comment]
Agreed. I'm working on getting this setup soon. I think GitHub Actions will be the way that we end up going. However, I've still been focused on documentation at the moment. So this is definitely in the back log for all of our libraries.

Which CI have you had the most success with?

**Comment by peter279k** [comment]
IMHO, the GitHub Actions is latest CI build and it's supported by GitHub directly and we can consider trying to integrate on this repository.

If possible, I can work on this issue and help this repository to integrate GitHub Actions :).

**Comment by peter279k** [comment]
Also, the most of people uses on GitHub repositories is Travis CI.

**Comment by joshualjohnson** [comment]
GitHub made it so easy, I did it in 10 minutes!

https://github.com/ua1-labs/firetest/pull/8
